### ISSUE TITLE
Improve organization of the data, labels and models needed in tutorials and readme.

### ISSUE THREAD

**Issue by elboyran** [dianna-ai/dianna#667]
There are three places for data, models, and labels in our repo:

1. https://github.com/dianna-ai/dianna/tree/main/tutorials/data (data and labels)
2. https://github.com/dianna-ai/dianna/tree/main/tutorials/models (models)
3. https://github.com/dianna-ai/dianna/tree/main/dianna/data (as visible in our updated schema, see #639: data, lables and models, some repetition of those in locations 1 and 2 above )

This is messy and we should improve on organizing the repo and remove the repetitoins!

**Comment by elboyran** [comment]
PR #699 sovles part of the issue (dashboard).
Tutorials remain to be fixed.

**Comment by elboyran** [comment]
Relocated all tutorials and the used by them: data, models and labels
TODO: restore bee.jpg!

Remaining: Update README with correct links, mention the new folders inclusind the ONNX conversions

**Comment by elboyran** [comment]
Old and proposed folder structures of `dianna/dianna `and `dianna/tutorials` :
---------------------
## `main` branch
### `dianna` (6 directories, 56 files)

├── __init__.py
├── ...
├── dashboard
│   ├── ...
├── data
│   ├── bee_2.png
│   ├── binary-mnist.npz
│   ├── digit0.jpeg
│   ├── digit0.jpg
│   ├── digit0.png
│   ├── digit1.png
│   ├── labels_mnist.txt
│   ├── labels_resnet50.txt
│   ├── labels_text.txt
│   ├── leafsnap_model.onnx
│   ├── logo.png
│   ├── mnist_model_tf.onnx
│   ├── movie_review_model.onnx
│   ├── movie_reviews_word_vectors.txt
│   ├── season_prediction_model_temp_max_binary.onnx
│   ├── weather_data.npy
│   └── weather_data_labels.txt
├── methods
│   ├── ...
├── utils
│   ├── ...
└── visualization
│   ├── ...

### `tutorials` (4 directories, 38 files)

├── README.md
├── conversion_onnx
│   ├── ...
├── data
│   ├── binary-mnist.npz
│   ├── coffee_test.csv
│   ├── coffee_train.csv
│   ├── leafsnap_classes.csv
│   ├── leafsnap_example_acer_rubrum.jpg
│   ├── movie_reviews_word_vectors.txt
│   └── shapes.npz
├── demo.ipynb
├── img
│   ├── bee.jpg
│   └── digit0.jpg
├── kernelshap_geometric_shapes.ipynb
├── kernelshap_mnist.ipynb
├── kernelshap_tabular_penguin.ipynb
├── kernelshap_tabular_weather.ipynb
├── lime_images.ipynb
├── lime_tabular_penguin.ipynb
├── lime_tabular_weather.ipynb
├── lime_text.ipynb
├── lime_timeseries_coffee.ipynb
├── lime_timeseries_weather.ipynb
├── models
│   ├── coffee.onnx
│   ├── geometric_shapes_model.onnx
│   ├── leafsnap_model.onnx
│   ├── mnist_model.onnx
│   ├── mnist_model_tf.onnx
│   ├── movie_review_model.onnx
│   ├── penguin_model.onnx
│   ├── season_prediction_model_temp_max_binary.onnx
│   └── sunshine_hours_regression_model.onnx
├── rise_imagenet.ipynb
├── rise_mnist.ipynb
├── rise_text.ipynb
└── rise_timeseries_weather.ipynb

----------------------------
## `667-reorganize`  branch
### `dianna` (8 directories, 66 files)
├── __init__.py
├── ...
├── dashboard
│   ├── ...
├── data
│   ├── bee.jpg
│   ├── binary-mnist.npz
│   ├── digit0.jpeg
│   ├── digit0.jpg
│   ├── digit0.png
│   ├── digit1.png
│   ├── leafsnap_example_acer_rubrum.jpg
│   ├── logo.png
│   ├── shapes.npz
│   └── weather_data.npy
├── labels
│   ├── coffee_test.csv
│   ├── coffee_train.csv
│   ├── labels_mnist.txt
│   ├── labels_resnet50.txt
│   ├── labels_text.txt
│   ├── leafsnap_classes.csv
│   ├── movie_reviews_word_vectors.txt
│   └── weather_data_labels.txt
├── methods
│   ├── ...
├── models
│   ├── coffee.onnx
│   ├── geometric_shapes_model.onnx
│   ├── leafsnap_model.onnx
│   ├── mnist_model.onnx
│   ├── mnist_model_tf.onnx
│   ├── movie_review_model.onnx
│   ├── penguin_model.onnx
│   ├── season_prediction_model_temp_max_binary.onnx
│   └── sunshine_hours_regression_model.onnx
├── utils
│   ├── ...
└── visualization
    ├── ...

### `tutorials` (5 directories, 20 files)

├── README.md
├── conversion_onnx
│   ├── keras2onnx.ipynb
│   ├── pytorch2onnx.ipynb
│   ├── skl2onnx.ipynb
│   └── tensorflow2onnx.ipynb
├── demo.ipynb
└── explainers
    ├── KernelSHAP
    │   ├── kernelshap_geometric_shapes.ipynb
    │   ├── kernelshap_mnist.ipynb
    │   ├── kernelshap_tabular_penguin.ipynb
    │   └── kernelshap_tabular_weather.ipynb
    ├── LIME
    │   ├── lime_images.ipynb
    │   ├── lime_tabular_penguin.ipynb
    │   ├── lime_tabular_weather.ipynb
    │   ├── lime_text.ipynb
    │   ├── lime_timeseries_coffee.ipynb
    │   └── lime_timeseries_weather.ipynb
    └── RISE
        ├── rise_imagenet.ipynb
        ├── rise_mnist.ipynb
        ├── rise_text.ipynb
        └── rise_timeseries_weather.ipynb
